server/server.js

```diff
@@ -97,14 +88,16 @@ class Server {
                 heartbeat: 2000
             }));
 
-            app.get('*', function response(req, res) {
+            app.get('*', (req, res) => {
                 var token = undefined;
 
                 if(req.user) {
                     token = jwt.sign(req.user, config.secret);
                 }
 
-                var html = pug.renderFile('views/index.pug', { basedir: path.join(__dirname, '..', 'views'), user: req.user, token: token });
+                let user = Util.getUserWithDefaultsSet(req.user);
```

Comment:
It might not even be necessary here, now that I've put it in deserialize, but hey you can't be too sure